.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: F69XcxcGJf5a8H0dFA57giP0ePIzq53nt
```

Comment:
Empty lines need to be added I expect these to have been corrected

Reply:
I'm sorry for this that was an oversight.